src/context/videos/index.js

```diff
@@ -0,0 +1,3 @@
+import { VideosProvider, useVideos } from "./Context";
+
+export { VideosProvider, useVideos };
```

Comment:
Why not export in a single line? 

`export { VideosProvider, useVideos } from './Context';`

Reply:
Not sure. I like to keep things separated.